cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -151,6 +156,95 @@ CREATE TABLE IF NOT EXISTS zipcodes (
 CREATE INDEX IF NOT EXISTS geom_index ON counties USING GIST (geom);
 CREATE UNIQUE INDEX IF NOT EXISTS zipcode_index ON zipcodes (zipcode);
 
+--- Returns lead connections aggregated by either state or water system, depending on z (zoom level).
+
+CREATE OR REPLACE FUNCTION public.lead_connections_function_source(z integer, x integer, y integer, query_params json) RETURNS bytea AS $$
+DECLARE
+    mvt bytea;
+BEGIN
+    IF (z < 6) THEN
+        -- Aggregate est. lead connections by state. Select relevant fields from both tables.
+        SELECT INTO mvt ST_AsMVT(tile,
+                                 'public.lead_connections_function_source',
+                                 4096, 'geom')
+        FROM (
+                 SELECT ST_AsMVTGeom(ST_Transform(s.geom, 3857),
+                                     ST_TileEnvelope(z, x, y)) AS geom,
+                        s.name                                 as state_name,
+                        count(w.lead_connections_count)        as lead_connections_count,
```

Comment:
Is it intentional to use `count` of counts here? It seems like we might want the `sum` of these columns, not the count of rows.

Reply:
Done.